integration-mc1-16/src/main/kotlin/org/eln2/ModBlocks.kt

```diff
@@ -3,14 +3,13 @@ package org.eln2
 import net.minecraft.block.Block
 import org.eln2.blocks.FlubberBlock
 import org.eln2.blocks.OreBlock
-import org.eln2.node.NodeBlock
 import org.eln2.utils.OreData
 
 /**
  * Blocks added here are automatically registered.
  */
 enum class ModBlocks(val block: Block) {
     FLUBBER(FlubberBlock()),
-    ORE_NATIVE_COPPER(OreBlock(OreData(1.0f, "ore_native_copper", ModItems.ORE_CHUNKS_COPPER.items.defaultInstance))),
-    NODE_BLOCK(NodeBlock())
+    ore_native_copper(OreBlock(OreData(1.0f, "ore_native_copper", ModItems.ore_chunks_copper.items.defaultInstance))),
```

Comment:
Enums are always caps

Reply:
Was dealing with an error about caps, I forgot to restore it to caps... Sorry, I'll patch that